src/blr/cli/init.py

```diff
@@ -0,0 +1,95 @@
+"""
+Create and initialize a new analysis directory.
+"""
+import logging
+import os
+import os.path
+import sys
+from pathlib import Path
+from importlib_resources import read_binary
+
+from ..utils import guess_paired_path
+
+logger = logging.getLogger(__name__)
+
+
+CONFIGURATION_FILE_NAME = "blr.yaml"
+
+
+def add_arguments(parser):
+    parser.add_argument(
+        "--reads1",
+        "--r1",
+        type=Path,
+        required=True,
+        metavar="READS",
+        help="First paired-end read file (.fastq.gz). The second is found automatically.",
+    )
+    parser.add_argument("directory", type=Path, help="New analysis directory to create")
+
+
+def main(args):
+    if args.reads1 is None:
+        logger.error("--reads1 must be given")
```

Comment:
With the change to the read1 argument we can remove this check as it does the same thing.

Reply:
done